bindata/network/kuryr/001-rbac.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kuryr
+rules:
+- apiGroups: [""]
+  resources:
+  - namespaces
+  - nodes
+  - endpoints
+  - services
+  - services/status
+  - pods
+  verbs: ["*"]
+- apiGroups: ["extensions"]
+  resources:
+  - networkpolicies
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: ["networking.k8s.io"]
+  resources:
+  - networkpolicies
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: ["k8s.cni.cncf.io"]
+  resources:
+  - network-attachment-definitions
+  verbs:
+  - get
+- apiGroups: ["openstack.org"]
+  resources:
+  - kuryrnets
+  - kuryrnetpolicies
+  verbs: ["*"]
+- apiGroups: ["oapi"]
+  resources:
+  - routes
+  verbs: ["*"]
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kuryr
+  namespace: kuryr
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kuryr
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kuryr
+subjects:
+- kind: ServiceAccount
+  name: kuryr
+  namespace: kuryr
+
+---
+# Rather than PATCHing the SecurityContextConstraints object, we can bind to 
+# it. This is undocumented, I'm copying it from openshift-sdn.
+# TODO: When the openshift-api control plane is splut out, we will need to
+# remove this and replace it with something (sigh) hardcoded
+apiVersion: authorization.openshift.io/v1
```

Comment:
This can now be removed!

Reply:
Done, can you tell my why this is no longer required?